nbi/buildspec.yml

```diff
@@ -30,8 +30,9 @@ phases:
       - mkdir -p envs
       - conda config --set path_conflict warn
       - conda env create --name $BRAKET_ENV -f environment.yml --no-default-package --solver libmamba
+      - pip cache purge
       - conda pack --quiet --name $BRAKET_ENV --output envs/Braket.tar --compress-level 1 --n-threads -1
-      - zstd -9 envs/Braket.tar -T0
+      - zstd -9 envs/Braket.tar -T0 --auto-threads=logical --rsyncable --sparse
```

Comment:
Hey, this is great. Thanks! I am wondering would `rysncable` would be beneficial since we are overwriting the entire s3 object anyways?